docs/reference/ml/delayed-data-detection.asciidoc

```diff
@@ -0,0 +1,41 @@
+[role="xpack"]
+[[ml-delayed-data-detection]]
+=== Working with delayed data
+
+In short, delayed data are any documents that are indexed *after* a datafeed has already gathered data for the 
+point in time referenced in the data time-field value.
```

Comment:
Thanks, that's clearer. I wonder if it could be simplified further like this:

_Delayed data_ are documents that are indexed late. That is to say, it is data
related to a time that the {dfeed} has already processed.